src/plugins/plan/plugin.js

```diff
@@ -21,34 +21,44 @@
  *****************************************************************************/
 
 import PlanViewProvider from './PlanViewProvider';
-import PlanInspectorViewProvider from "./inspector/PlanInspectorViewProvider";
+import ActivityInspectorViewProvider from "./inspector/ActivityInspectorViewProvider";
+import GanttChartInspectorViewProvider from "./inspector/GanttChartInspectorViewProvider";
+import ganttChartCompositionPolicy from './GanttChartCompositionPolicy';
 
 export default function (configuration) {
     return function install(openmct) {
         openmct.types.addType('plan', {
             name: 'Plan',
             key: 'plan',
+            description: 'A non-configurable timeline-like view for a compatible mission plan file.',
+            creatable: false,
+            cssClass: 'icon-plan',
+            form: [],
+            initialize: function (domainObject) {
+                domainObject.configuration = {
+                    clipActivityNames: false
```

Comment:
Let's reuse the default const from the PlanViewConfiguration class instead of assigning to `false` here.